catalog/nsili/catalog-nsili-endpoint/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -47,12 +47,21 @@
             <array/>
         </property>
         <property name="corbaOrb" ref="nsiliCorbaOrb" />
+        <property name="emailSender" ref="emailSender" />
     </bean>
 
     <bean id="nsiliWebSvc" class="org.codice.alliance.nsili.endpoint.NsiliWebEndpoint">
         <argument ref="nsiliEndpoint"/>
     </bean>
 
+
+    <bean id="emailSender"
+          class="org.codice.alliance.nsili.endpoint.EmailSender">
+
+        <argument value="donotreply@test.com"/>
+        <argument value="connexta-com.mail.protection.outlook.com"/>
```

Comment:
We may want to use something related to Codice here, and if that's not possible leave it blank